kiali/README.md

```diff
@@ -40,22 +40,38 @@ An alternative can be a parallel installation of the upstream chart offering all
 
 1. Run the Helm upgrade command, which installs the chart if not present yet.
     ```bash
-    helm upgrade --install --create-namespace -n ${KYMA_KIALI_NS} ${HELM_RELEASE_NAME} kiali/kiali-operator --set cr.spec.auth.strategy=anonymous -f https://raw.githubusercontent.com/kyma-project/examples/main/kiali/values.yaml
+    helm upgrade --install --create-namespace -n ${KYMA_KIALI_NS} ${HELM_RELEASE_NAME} kiali/kiali-operator -f https://raw.githubusercontent.com/kyma-project/examples/main/kiali/values.yaml
```

Comment:
Pardon me, I forgot that it works with or without the curly brackets when I added the comment. It must have been because of the other error I got that I just connected the wrong dots. 
But either way, without the brackets is the convention we use in Kyma documentation for environment variables, and we use the brackets for placeholders, so it's still good that you removed them. I just wanted to correct myself :)